### PR TITLE
finance.yahoo.co.jp

### LINKED ISSUE #125562: finance.yahoo.co.jp
### Issue URL (Social Widget)
 [https://finance.yahoo.co.jp/quote/%5EDJI](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ffinance.yahoo.co.jp%2Fquote%2F%255EDJI)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.4.0
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

Fix #125562 

### Add your comment and screenshots

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SocialFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -102,6 +102,7 @@ journals.lww.com#?#.ejp-article-tools__list > .ejp-article-tools__list-item:has(
 enoteca.co.jp#?#.justify-end > .md\:text-left:contains(/^この記事をシェア$/)
 ichijin-plus.com#?#main > div[class^="sc-"][class*="sc-"] > div[class^="sc-"]:has(> div[class] a[href^="https://twitter.com/intent/tweet?"])
 creators.yahoo.co.jp#?#.ArticleDetail__actionTitle:contains(シェア)
+finance.yahoo.co.jp#?#div[class^="_"]:has(> ul > li > a[href="#"][target="_blank"])
```
Comment (zloyden) [inline SocialFilter/sections/css_extended.txt:105]: @SKEIDs please check this on your end
```
finance.yahoo.co.jp#?#div[class^="_"]:has(> ul > li > a[href="#"][target="_blank"][aria-label^="Facebook"])
```

Comment (Yuki2718) [reply SocialFilter/sections/css_extended.txt:105]: But I don't like `#?#div[class^="_"]:has`, too many nodes to evalutate.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Change to `finance.yahoo.co.jp##header + ul > li > a[class][aria-label*="で"]` and move to specific.txt, no need to use extended syntax.

File: AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt [hunk 2]
```diff
@@ -24,7 +24,7 @@ flightradar24.com##.overlay-open-in-app-button
 giphy.com##div[class*="OpenInAppContainer"]
 search.yahoo.co.jp##div[class^="AppPromo"]
 search.yahoo.co.jp##.W2ACarousel_W2ACarousel__ggrnX
-finance.yahoo.co.jp#?#div[class^="_"] > a[href="https://app.adjust.com/nr7n4gu"]:upward(1)
+finance.yahoo.co.jp#?#div[class^="_"] > a[href^="https://app.adjust.com/"]:upward(1)
```
Comment (Yuki2718) [inline AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt:27]: Apparently no need for `:upward()` here too.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

**Comment by SKEIDs** [comment]
I see.
I will fix all.

**Review by Yuki2718 — APPROVED** [review]

**Comment by Yuki2718** [comment]
BTW on my end `/nr7n4gu` still hits, tho I don't see FP by removing this.

**Comment by SKEIDs** [comment]
> BTW on my end `/nr7n4gu` still hits, tho I don't see FP by removing this.

should I change `a[href^="https://app.adjust.com/"] ` to `a[href="https://app.adjust.com/"]`?

**Comment by Yuki2718** [comment]
If you mean `a[href="https://app.adjust.com/nr7n4gu"]`, maybe no need. If FP is ever reported we can revert it back.

**Comment by SKEIDs** [comment]
I checked [#124942](https://github.com/AdguardTeam/AdguardFilters/issues/), but i can not find `/nr7n4gu`.

> If FP is ever reported we can revert it back.

I agree with this.

**Review by zloyden — APPROVED** [review]

**Pull request merged**